src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -146,7 +146,16 @@ private function parseDefinitions(\DOMDocument $xml, $file, $defaults)
         foreach ($services as $service) {
             if (null !== $definition = $this->parseDefinition($service, $file, $defaults)) {
                 if ('prototype' === $service->tagName) {
-                    $this->registerClasses($definition, (string) $service->getAttribute('namespace'), (string) $service->getAttribute('resource'), (string) $service->getAttribute('exclude'));
+                    $excludes = array_map(function ($node) { return $node->nodeValue; }, $this->getChildren($service, 'exclude'));
```

Comment:
would array_column work here instead of array_map?

Reply:
nope. We need a property access, not an array access